islands/files/FilesWrapper.tsx

```diff
@@ -7,6 +7,7 @@ interface FilesWrapperProps {
   initialPath: string;
   baseUrl: string;
   isFileSharingAllowed: boolean;
+  isDirectoryDownloadsAllowed: boolean;
```

Comment:
Same as above, please rename (applies to below as well, I won't mention again so it's not a bother).